draft-ietf-quic-tls.md

```diff
@@ -745,6 +745,9 @@ handled separately.
 sample_offset = 6 + len(destination_connection_id) +
                     len(source_connection_id) +
                     len(payload_length) + 4
+if packet_type == Initial:
+    sample_offset += len(token_length) +
+                     len(token)
 ~~~
 
```

Comment:
Unless I am missing something, len(token_length) is only available *after* the PN has been decrypted.

Reply:
So we have a circular dependency when decrypting. See issue #1535.

Reply:
My bad. This is actually addressed later, as the token has now moved in front of the PN.